src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -245,6 +256,21 @@ public AzureAppConfigurationOptions TrimKeyPrefix(string prefix)
             return this;
         }
 
+        /// <summary>
+        /// Trims the provided prefix from the keys of all feature flags retrieved from Azure App Configuration.
+        /// </summary>
+        /// <param name="prefix">The prefix to be trimmed from all feature flags retrieved from Azure App Configuration.</param>
```

Comment:
>The prefix to be trimmed from all feature flags retrieved from Azure App Configuration. [](start = 33, length = 87)

This could be shortened to `The prefix to be trimmed.` to avoid repeating the details in the summary comment.